schemas/property_assignment.py

```diff
@@ -0,0 +1,32 @@
+from ma import ma
+from models.property_assignment import PropertyAssignment
+from models.user import UserModel, RoleEnum
+from models.property import PropertyModel
+from marshmallow import fields, validates, ValidationError
+
+
+class PropertyAssignSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = PropertyAssignment
+        include_fk = True
+        relations = True
+
+        propertyID = fields.Nested('PropertySchema')
+        managerID = fields.Nested('UserSchema', required=True)
```

Comment:
I don't think these three lines are necessary. Are they being used for something I'm not seeing?

Reply:
included the nested fields to define The relationship in the schema. So I defined it there, Otherwise the Auto schema defines the fields as just Integer fields. I can do an inspection to see if they are just redundant.

Reply:
Similar pattern was used in Schema/leases.py

Reply:
I remember doing it in the lease to specify how to serialized the data, I don't recall if it was needed for another reason though. No action needed. The tests look good so let's get this merged.